package.json

```diff
@@ -77,15 +79,16 @@
     "json": "^11.0.0",
     "markdownlint-cli2": "^0.4.0",
     "postcss": "^8.4.47",
-    "prettier": "^2.7.1",
+    "prettier": "^3.3.3",
     "react": "^17.0.2",
     "react-docgen-typescript": "^2.1.0",
     "react-dom": "^17.0.2",
     "react-helmet": "^6.1.0",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5.0.0",
+    "react-syntax-highlighter": "^15.6.1",
```

Comment:
This seems like an addition. Next time I would appreciate a mention in the commit about why this is necessary - a bit like what you did for `prettier`.